scalopus_general/src/endpoint_manager_poll.cpp

```diff
@@ -70,48 +70,53 @@ void EndpointManagerPoll::manage()
   const auto providers = factory_->discover();
   for (const auto& destination : providers)
   {
-    if (transports_.find(destination->hash_code()) != transports_.end())
```

Comment:
Diff looks ugly, but it's a pretty boring change, I lifted everything in this for loop into its own function that takes a destination. This allows explicitly connecting to a single destination if you know the address, which we do if we make the transport server in our own process.